rs/nns/governance/src/neuron/types.rs

```diff
@@ -1197,7 +1206,9 @@ impl From<Neuron> for NeuronProto {
             neuron_type,
             visibility,
             voting_power_refreshed_timestamp_seconds,
-            recent_ballots_next_entry_index: recent_ballots_next_entry_index.map(|x| x as u32),
+            recent_ballots_next_entry_index,
+            deciding_voting_power,
+            potential_voting_power,
```

Comment:
This is where the new fields are populated.

There is another transcription step in conversions.rs to go from Prost-generated Rust (in the gen directory) to hand-crafted Rust (in api).